appserver/payara-appserver-modules/payara-jsr107/pom.xml

```diff
@@ -28,13 +28,16 @@
             <groupId>fish.payara.payara-modules</groupId>
             <artifactId>hazelcast-bootstrap</artifactId>
             <version>${project.version}</version>
-            <type>jar</type>
         </dependency>
         <dependency>
             <groupId>javax</groupId>
             <artifactId>javaee-api</artifactId>
             <version>7.0</version>
-            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>fish.payara</groupId>
+            <artifactId>payara-api</artifactId>
+            <version>4.1.1.171-SNAPSHOT</version>
```

Comment:
Could you change this to `${project.version}` please? It's nice to keep consistency, and makes keeping versions in line easier